shm/CMakeLists.txt

```
@@ -53,7 +53,8 @@ enable_testing()
 
 add_library(${PROJECT_NAME}-lib STATIC ${SRC})
 add_executable(${PROJECT_NAME} main.cpp)
-add_executable(${PROJECT_NAME}-ut test.cpp)
+add_executable(${PROJECT_NAME}-ut testShip.cpp)
+add_executable(${PROJECT_NAME}-ut testItem.cpp)
```

Does it work well? I have never tried adding multiple executables with the same name. Normally it should be like this:
```suggestion
add_executable(${PROJECT_NAME}-ut testShip.cpp testItem.cpp)
```